portal/eproms/views.py

```diff
@@ -118,7 +118,9 @@ def assessment_engine_view(user):
     comp_date = (
         localize_datetime(assessment_status.completed_date, user)
         if assessment_status.completed_date else None)
-    assessment_is_due = research_study_status[BASE_RS_ID]['ready']
+    assessment_is_due = (
+        BASE_RS_ID in research_study_status and
+        research_study_status[BASE_RS_ID]['ready'])
```

Comment:
chaining `.get()` is a little more succinct
```suggestion
    assessment_is_due = research_study_status.get(BASE_RS_ID, {}).get('ready', False)
```

Reply:
thank you, ivan, fixed.